.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: c
 script:
-  - ./autogen.sh
-  - ./configure
-  - make distcheck
+- ./autogen.sh
+- ./configure
+- make distcheck
+os:
+- linux
+- osx
+compiler:
+- gcc
+- clang
```

Comment:
FWIW, I used a single entry here on purpose. I don't expect any platform differences between osx and linux or between clang and gcc for this library (msvc and warnings from newer compiler versions are a different story...) so it didn't seem worthwhile to spend travis resources and slow down patch feedback.